integrations/GA4/utils.js

```diff
@@ -236,6 +236,13 @@ const getGa4SessionId = (measurementId) => {
     sessionId = Cookie.get(`_ga_${measurementIdArr[1]}`).split(".");
     return sessionId[2];
   }
+  if (
+    window.localStorage &&
+    window.localStorage.getItem(`_ga_${measurementIdArr[1]}`)
+  ) {
+    sessionId = window.localStorage.getItem(`_ga_${measurementIdArr[1]}`);
+    return sessionId[2];
+  }
```

Comment:
Use the storage module instead.

Reply:
@saikumarrs As we discussed before, Storage.getItem() function will throw an error because we are parsing response. is there any other function that you are referring to?

`SyntaxError: Unexpected token 'G', "GS1.1.1669"... is not valid JSON`

Reply:
updated